docs/sandbox/VehicleRentalServiceDirectory.md

```diff
@@ -38,6 +39,7 @@ the `router-config.json`
 | url                                                 |      `uri`      | Endpoint for the VehicleRentalServiceDirectory                                  | *Required* |               |  2.1  |
 | [headers](#vehicleRentalServiceDirectory_headers)   | `map of string` | HTTP headers to add to the request. Any header key, value can be inserted.      | *Optional* |               |  2.1  |
 | [networks](#vehicleRentalServiceDirectory_networks) |    `object[]`   | List all networks to include. Use "network": "default-network" to set defaults. | *Optional* |               |  2.4  |
+|       allowKeepingVehicleAtDestination              |    `boolean`    | Enables `allowKeepingVehicleAtDestination` for the given network                | *Optional* | `false`       |  2.5  |
```

Comment:
Not sure if we should give a more thorough explanation for this param here.

Reply:
I can copy the longer version from the GBFS documentation? Or are you looking for something in the middle?

Reply:
Either works.